sda-commons-dependencies/build.gradle

```diff
@@ -100,7 +101,11 @@ dependencies {
 
     // sda-commons-server-auth
     api "com.auth0:java-jwt:4.4.0"
-    api "org.bouncycastle:bcpkix-jdk15on:1.70" // use the same version as bcprov-jdk15on in DW
+    api "org.bouncycastle:bcpkix-jdk18on:1.76" // use the same version as bcprov-jdk18on
```

Comment:
I guess it's important and makes sense to always use the same versions of the Bbouncycastle library. Actually that is what we verify in module [sda-commons-dependency-check](https://github.com/SDA-SE/sda-dropwizard-commons/blob/master/sda-commons-dependency-check/build.gradle#L48).